azure-out-of-cluster.md

```diff
@@ -0,0 +1,98 @@
+
+## Step 1: Export Azure Cost Report
+
+Follow this guide taking note of the name that you use for the exported cost report and select the daily month-to-date option for how the reports will be created.
+
+https://docs.microsoft.com/en-us/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal
+
+It will take a few hours to generate the first report, after which Kubecost can use the Azure Storage API to pull that data. 
+
+>note: If you have sensitive data in an Azure Storage account and do not want to give out access to it, create a separate Azure Storage account to store your cost data export.
+
+## Step 2: Provide Access to Azure Storage API
+
+The values needed to provide access to the Azure Storage Account where cost data is being exported can be found in the Azure portal in the Storage account where the cost data is being exported. 
+* `<STORAGE_ACCOUNT_NAME>` is the name of the Storage account where the exported CSV is being stored.
+* `<STORE_ACCESS_KEY>` can be found by selecting the “Access Keys” option from the navigation sidebar  then selecting “Show Keys”. Using either of the two keys will work. 
+* `<REPORT_CONTAINER_NAME>` is the name that you choose for the exported cost report when you set it up. This is the name of the container where the CSV cost reports are saved in your Storage account. 
+
+<details>
+	<summary>Maually add secret to cluster (Recommended)</summary>
+	To create this secret you will need to create a JSON file that must be named azure-storage-config.json
+ with the following format:
+
+```
+{
+	"azureStorageAccount": "<STORAGE_ACCOUNT_NAME>",
+	"azureStorageAccessKey": "<STORE_ACCESS_KEY>",
+	"azureStorageContainer": <REPORT_CONTAINER_NAME>
+}
+```
+
+Once you have the values filled out use this command to create the secret:
+
+`kubectl create secret generic <SECRET_NAME> --from-file=azure-storage-config.json -n kubecost`
+
+Once the secret is created, set .Values.kubecostProductConfigs.azureStorageSecretName to
+<SECRET_NAME> and upgrade Kubecost via helm, other values related to Azure Storage should not be set.
+</details>
+<br>
+ <details>
+		<summary>Create a secret from helm values</summary>
+
+* Set `.Values.kubecostProductConfigs.azureStorageAccount = <STORAGE_ACCOUNT_NAME>`
+* Set `.Values.kubecostProductConfigs.azureStorageAccessKey = <STORE_ACCESS_KEY>`
+* Set `.Values.kubecostProductConfigs.azureStorageContainer = <REPORT_CONTAINER_NAME>`
+* Set `.Values.kubecostProductConfigs.azureStorageCreateSecret = true`
+* Do not set `.Values.kubecostProductConfigs.azureStorageSecretName` if you are using this approach
+
+> Note: that this will leave your secrets unencrypted in values.yaml. Use an existing secret as in the next step to avoid this.
```

Comment:
```suggestion
> Note: that this will leave your secrets unencrypted in values.yaml. Use a Kubernetes secret as in the previous method to avoid this.
```